### PR TITLE
Add multi-node tutorial

### PR DESCRIPTION
**Pull request by pmatulis**
Add a tutorial on setting up a multi-node cluster.

Make adjustments to the existing single-node cluster tutorial to maintain consistency.

See PR  #200 for context.

### CODE REVIEW

File: docs/tutorial-2/multi-node.rst [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+   sudo snap install microceph
+
+Prevent the software from being auto-updated:
+
+.. code-block:: none
+
+   sudo snap refresh --hold microceph
+
+Repeat the above three steps for node-2 and node-3.
```
Comment (sabaini) [inline docs/tutorial-2/multi-node.rst:54]: There's two steps not three -- snap install and hold refresh -- right?

File: docs/tutorial-2/multi-node.rst [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+Prepare the cluster
+-------------------
+
+On **node-1** we will now:
+
+* initialis the cluster
```
Comment (sabaini) [inline docs/tutorial-2/multi-node.rst:61]: Typo: initialize

File: docs/tutorial-2/multi-node.rst [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+* create registration tokens
+
+Initialise the cluster with the :command:`cluster bootstrap` command:
```
Comment (sabaini) [inline docs/tutorial-2/multi-node.rst:64]: Typo: initialize

File: docs/tutorial-2/multi-node.rst [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+.. code-block:: none
+
+   sudo microceph cluster bootstrap
+
+Tokens are needed to join the other two nodes to the cluster. Generate these
+with the :command:`cluster add` command.
```
Comment (sabaini) [inline docs/tutorial-2/multi-node.rst:71]: Just for avoidance of doubt, could maybe add a note that the tokens will be different on the users' machine

File: docs/tutorial-2/multi-node.rst [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+found in the :doc:`Howto <../how-to/index>` section.
+
+The cluster can also be managed using native Ceph tooling if snap-level
+commands are not yet available for a desired task by appending a native command
+to the :command:`microceph` command. This is the equivalent to the standard
+:command:`ceph status` command for instance:
+
+.. code-block:: none
+
+   microceph.ceph status
```
Comment (sabaini) [inline docs/tutorial-2/multi-node.rst:166]: We have now aliases in for the high-level ceph cli commands -- e.g. `ceph`, `rbd`, `radosgw-admin` all point to `microceph.ceph`, `microceph.rbd` etc. 
So from my perspective we don't need to mention the `microceph.` prefixed commands here

File: docs/tutorial-2/multi-node.rst [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+       osd: 3 osds: 3 up (since 4s), 3 in (since 6s)
+
+     data:
+       pools:   1 pools, 1 pgs
+       objects: 0 objects, 0 B
+       usage:   336 MiB used, 60 GiB / 60 GiB avail
+       pgs:     100.000% pgs unknown
+                1 unknown
+
+Naturally you are free to use Ceph commands directly: :command:`sudo ceph
```
Comment (sabaini) [inline docs/tutorial-2/multi-node.rst:188]: See above, due to the aliasing `ceph` and `microceph.ceph` are the same binary

File: docs/tutorial-2/single-node.rst [hunk 7]
```diff
@@ -125,12 +128,13 @@ Your Ceph cluster is now deployed and can be managed by following the resources
...
 The cluster can also be managed using native Ceph tooling if snap-level
-commands are not yet available for a desired task. This is the standard
+commands are not yet available for a desired task by appending a native command
+to the :command:`microceph` command. This is the equivalent to the standard
 :command:`ceph status` command for instance:
 
 .. code-block:: none
 
-   ceph status
+   microceph.ceph status
```
Comment (sabaini) [inline docs/tutorial-2/single-node.rst:137]: See above -- we have now aliases in for the high-level ceph cli commands

**Review by sabaini — CHANGES_REQUESTED** [review]
Hi, thanks for this addition -- some inline commentary

File: docs/tutorial-2/multi-node.rst (hunk 2 above)
Comment (pmatulis) [reply docs/tutorial-2/multi-node.rst:61]: But the "ise" ending is perfectly good UK English, which is the standard we use in all Canonical documentation.

File: docs/tutorial-2/multi-node.rst (hunk 3 above)
Comment (pmatulis) [reply docs/tutorial-2/multi-node.rst:64]: ditto - This spelling is ok.

File: docs/tutorial-2/multi-node.rst (hunk 5 above)
Comment (pmatulis) [reply docs/tutorial-2/multi-node.rst:166]: Interesting ok!

**Review by sabaini — APPROVED** [review]

**Pull request merged**